parsec/backend/memory/user.py

```diff
@@ -285,26 +275,33 @@ async def find_humans(
             )
             for user in data
         ]
-
+        results = sorted(
```

Comment:
A comment should be added here to explain the reason (but I think this code is not needed, see my other comment below)